### PR TITLE
Assignment one completed

### PR DESCRIPTION
**Pull request by chrisstafford**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,12 +7,35 @@
...
+        <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <version>3.1</version>
+              <configuration>
+                <compilerArgument>-Xlint</compilerArgument>
+                <encoding>${project.build.sourceEncoding}</encoding>
+                <source>1.7</source>
+                <target>1.7</target>
```
Comment (smarks) [inline pom.xml:26]: this should probably be 1.8

**Comment by smarks** [comment]
Excellent work. Maybe listen to this week's chat to get a better sense of dependency injection.

**Comment by smarks** [comment]
where is the ciricle ci continuous integration part?
I'll hold off the final grade until I see that.

**Comment by chrisstafford** [comment]
> where is the ciricle ci continuous integration part?
> I'll hold off the final grade until I see that.

I tried to add CircleCI but github was giving me an error about my billing details. I posted about it in the discussion board, but it seems I'm the only one having issues. I'll do some troubleshooting tonight and see what I can do.

**Comment by smarks** [comment]
Sounds good

**Comment by chrisstafford** [comment]
This should be good for final review.

**Pull request merged**